### ISSUE TITLE
Error Message When Saving Schedule using iCal (Mac Calendar)

### ISSUE THREAD

**Issue by baozi90** [ComputerScienceHouse/schedulemaker#79]
I keep getting an error message (screen shot attached) in the Mac Calendar after saving the schedule to iCal. It seems to pop up each time when I either first saved the schedule or try to make a change to any part of the calendar. 

This issue may only or more related to the iCloud Calendar. That is, the error message seems to pop up only when I saved to the iCloud Calendars and none when saved to the "RIT" Calendar. 

RIT Calendar - RIT google account calendar that's synced to the Mac Calendar. 

![screen shot 2015-04-08 at 22 17 38](https://cloud.githubusercontent.com/assets/11864006/7059631/1d8930cc-de42-11e4-9d40-d2f567e29ee7.png)

**Comment by bgrawi** [comment]
I am pretty sure this does not have to do with the iCal file generated by Schedule Maker, but I can test this later on a mac if by some weird way it is. This looks like there is a problem authorizing your Calendar app to iCloud. Can you add other events to the calendar just fine?

**Comment by baozi90** [comment]
Yes, when adding to calendar other than iCloud, there's no error message(s) popping up. Instead, when saving the schedules to iCloud, there are error messages popping up depending on however many schedules I've added. I can't test it out correctly just yet (because I still need to keep those schedules for now), but there doesn't seem to be error message(s) when I manually add events to iCloud calendar. 

This may be another issue, but as I tried to test it out, some creates a duplicate of the same class of the same time and day.
